src/darwin/Framework/CHIP/MTROperationalBrowser.mm

```diff
@@ -107,7 +117,10 @@
     }
 
     ChipLogProgress(Controller, "Notifying controller factory about new operational instance: '%s'", aName);
-    [self->mDeviceControllerFactory operationalInstanceAdded:peerId];
+    MTRDeviceControllerFactory * strongFactory = self->mDeviceControllerFactory;
```

Comment:
The old code did exactly the same thing, calling the message on a nil mDeviceControllerFactory is a no-op anyway.

Reply:
Will undo this change.